ivy/functional/backends/torch/layers.py

```diff
@@ -284,7 +284,9 @@ def depthwise_conv2d(
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
     strides = [strides] * 2 if isinstance(strides, int) else strides
+    strides = [strides[1], strides[2]] if len(strides) == 4 else strides
     dilations = [dilations] * 2 if isinstance(dilations, int) else dilations
+    filters = torch.squeeze(filters, 3) if filters.ndim == 4 else filters
```

Comment:
This could be a reason for problem. Can you try changing the `torch.squeeze` to `ivy.squeeze`? IYou will first need to check if we have this function implemented already.

Reply:
@kurshakuz , thanks for all the patience with reviews !!

Yes, I knew torch.squeeze was causing error, however got confused since there were no errors through pytest, but testing manually was showing error.

- ivy.squeeze does exist, but then using this was also throwing errors. 
- For some reason, x, filters were passed to the backend as ndarrays and not torch tensors. This was causing the issue, tyecasting helped.

x = ivy.array([[[[0.5 ]], [[0.5 ]], [[0.5 ]], [[0.75]]]]).astype(ivy.float32)
filters = ivy.array([[[[0.5]]], [[[0.5]]], [[[0.5]]]]).astype(ivy.float32)
padding = 'VALID'
dilations = [1, 1]
strides = [1, 2, 2, 1]
ivy.set_backend('torch')
y_torch = ivy.functional.frontends.tensorflow.nn.depthwise_conv2d(x, filters, strides, padding, dilations=dilations)
y_torch
ivy.frontends.tensorflow.EagerTensor([[[[0.75]]]],shape=(1, 1, 1, 1),dtype=float32)

Hope, it is all in place and this is one last time. Please review.